src/www/ui/admin-license-file.php

```diff
@@ -103,11 +104,26 @@ public function Output()
 
     $V .= $this->Inputfm();
     if (@$_POST["req_shortname"])
+    {
       $V .= $this->LicenseList($_POST["req_shortname"], $_POST["req_marydone"]);
-
+      $V .= '<script src="scripts/tools.js"></script>';
+      $V .= $this->getDataTableScript();
+    }
     return $V;
   }
 
+  function getDataTableScript()
+  {
+    return <<<EOT
+    <script src="scripts/jquery.dataTables.min.js" type="text/javascript"></script>
```

Comment:
It looks like that `css/jquery.dataTables.css` is loaded by default but `scripts/jquery.dataTables.min.js` is added on demand? That looks inconsistend (but is not a problem of this PR).